### DISCUSSION TITLE
Database socket path issue with GCP [Oct 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4184] [Discord Threads]
> 🙋‍♂️ **paxon**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### <@869449535733239869>  
#  
> 👨‍💻 **pranavxc**  
#### Hi <@611240793654427687> can you specify `host` as the `socketPath` and try it ?    
####    
#### From google cloud docs I found following sample connection code   
####    
#### ```js   
####  return Knex({   
####     client: 'pg',   
####     connection: {   
####       user: process.env.DB_USER, // e.g. 'my-user'   
####       password: process.env.DB_PASS, // e.g. 'my-user-password'   
####       database: process.env.DB_NAME, // e.g. 'my-database'   
####       host: process.env.INSTANCE_UNIX_SOCKET, // e.g. '/cloudsql/project:region:instance'   
####     },   
####     // ... Specify additional properties here.   
####     ...config,   
####   });   
#### ```   
####    
#### Ref : https://cloud.google.com/sql/docs/postgres/samples/cloud-sql-postgres-knex-connect-unix  
#  
> 🙋‍♂️ **paxon**  
#### <@682854952657616902> That worked! Thank you so much  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1029512702399365212)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
